### ISSUE TITLE
window event listeners conflicts with player

### ISSUE THREAD

**Issue by Robthegod23** [SevenOutman/vue-aplayer#57]
I have the following **scroll** event listener added to my component upon its creation:
```
created () {
  window.addEventListener('scroll', this.getHeight);
},
```
This function adjust the ```min-height``` of a certain container div
```
getHeight () {
   let w = window.innerHeight
   let t = this.$refs.container.getBoundingClientRect().top
   return this.containerHeight = w - t
},
```

however, when I initially play a song, then scroll the page, the player completely stops. Upon inspection, I see some weird behavior happening to the ```audio``` elements ```src``` attribute, as if its being reset with every fire of the scroll event:

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/13503681/40639818-0887de0e-62e1-11e8-8c3d-7b9decbec2ee.gif)

as you can see, whenever the ```min-height``` of the ```container``` div is changed, the players src is being affected...

**Comment by SevenOutman** [comment]
@Robthegod23 I guess you are using **object literal** as the `music` prop's value like:
```vue
<aplayer :music="{ src: 'an url' }" />
```
insteand of using a reference
```vue
<aplayer :music="myMusic" />
```

There's not many differences but one: every time the `<aplayer>` or its parents somehow get "refreshed", `music` is accepting a new object if you use object literals. And that can cause what you're observing now.

**Comment by Robthegod23** [comment]
Yes that was it. I was using an inline literal rather than a reference. Thanks mate!
